packages/main/src/StandardListItem.ts

```diff
@@ -170,9 +145,6 @@ class StandardListItem extends ListItem implements IAccessibleListItem {
 	 * <br><br>
 	 * <b>Note:</b> Although this slot accepts HTML Elements, it is strongly recommended that you only use text in order to preserve the intended design.
 	 *
-	 * @type {Node[]}
-	 * @name sap.ui.webc.main.StandardListItem.prototype.default
-	 * @slot
 	 * @public
```

Comment:
this slot has to be defined into class comment

Reply:
Don't know how to answer to your above comment, but I didn't remove ListItemBase from other file since they are not part of this change.